aws-datastore/src/main/java/com/amplifyframework/datastore/storage/sqlite/SQLiteCommandFactory.java

```diff
@@ -275,22 +274,18 @@ public SqlCommand insertFor(@NonNull ModelSchema modelSchema) {
             final String columnName = columnsIterator.next().getName();
             stringBuilder.append(Wrap.inBackticks(columnName));
             if (columnsIterator.hasNext()) {
-                stringBuilder.append(",").append(SqlKeyword.DELIMITER);
+                stringBuilder.append(SqlKeyword.SEPARATOR);
             }
         }
         stringBuilder.append(")")
                 .append(SqlKeyword.DELIMITER)
                 .append("VALUES")
                 .append(SqlKeyword.DELIMITER)
-                .append("(");
-        for (int i = 0; i < columns.size(); i++) {
-            if (i == columns.size() - 1) {
-                stringBuilder.append("?");
-            } else {
-                stringBuilder.append("?, ");
-            }
-        }
-        stringBuilder.append(")");
+                .append("(")
+                .append(Collections.nCopies(columns.size(), SqlKeyword.VARIABLE.toString())
```

Comment:
> I resolved it by simply using ID strings directly without binding them, which could potentially be a problem.
As of right now, ID cannot be anything than UUID format, so there is no concern for SQL injection. However, there is a conversation floating around in favor of allowing any string to be assignable to ID field (like iOS does), so I'm curious if this change would potentially introduce a vulnerability.

If you are no longer binding ID strings, why are you still creating this prepared statement with variables?  Quite likely, I just don't fully understand something here.

Reply:
oh oops. none of the changes in `SQLiteCommandFactory` is relevant for this PR. I will remove them for clarity.